versions.json

```diff
@@ -167,6 +167,23 @@
         ],
         "fpm_directories": ["include","lib"]
     },
+    "jwt-cpp": {
+        "enable_sha": true,
```

Comment:
so i think this is not necessary since we now have a fork in the irods namespace?    please try without it.

Reply:
This is needed because we're not using a tag or branch, but a commit sha to identify what to build.

Reply:
oh, i see.   it's not a 'real' tag by them.   it's pre-0.7.0.